addons/mod_tool/plugin.gd

```diff
@@ -1,7 +1,7 @@
 tool
 extends EditorPlugin
 
-var tools_panel: ModToolsPanel
+var tools_panel # ModToolsPanel (not typed to avoid issues with the store when disabled)
```

Comment:
That will, at least, allow us to enable the plugin without manually adding the autoload first. However, I think we have to rework the entire store because of #72.

Reply:
yep, temporary fix